### PR TITLE
Image similarity widget

### PR DESCRIPTION
**Pull request by PatrickBue**
### Description
Added interactive widget for image similarity

### Checklist:
- [x] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [x] I have added tests covering my contributions.
- [x] I have updated the documentation accordingly.
- [x] This PR is being made to `staging` and not `master`
- [x] I will squash merge this PR into `staging`

### CODE REVIEW

**Comment by PatrickBue** [comment]
@jiata can you please review

File: utils_cv/similarity/widget.py [hunk 1]
```diff
@@ -14,160 +15,131 @@ def _list_sort(list1D, reverse=False, comparison_fct=lambda x: x):
...
-                    children=[
-                        w_header,
-                        widgets.HBox(children=[self.w_img, w_info]),
-                    ]
-                )
-            ]
-        )
-        self.ui.set_title(0, "Results viewer")
+        # Create tab
+        self.ui = widgets.Tab(children=[w_img_grid])
```
Comment (jiata) [inline utils_cv/similarity/widget.py:141]: does this need to be a tab?

Comment (PatrickBue) [reply utils_cv/similarity/widget.py:141]: No, but it looks nicer with tab :)

**Comment by PatrickBue** [comment]
@jiata thanks for the feedback. Agree that the yellow button is not ideal, much better would be if we could click on the image itself (but the image widget class doesn't have a click callback). I'll leave it as is for now, and maybe leave it for later should we ever want to improve the UI.

**Pull request merged**